lib/src/components/form/form.dart

```diff
@@ -1349,12 +1349,22 @@ class FormField<T> extends StatelessWidget {
   final Widget label;
   final Widget? hint;
   final Widget child;
+  final Widget? leadingLabel;
+  final Widget? trailingLabel;
+  final MainAxisAlignment? labelAxisAlignment;
+  final Gap? leadingGap;
+  final EdgeInsetsGeometry? padding;
   final Validator<T>? validator;
 
   const FormField({
     required FormKey<T> super.key,
     required this.label,
     required this.child,
+    this.leadingLabel,
+    this.trailingLabel,
+    this.labelAxisAlignment = MainAxisAlignment.spaceBetween,
+    this.leadingGap = const Gap(5),
```

Comment:
What's the benefit of passing the Gap widget itself instead of the gap value (double) here?